README.md

```diff
@@ -1,41 +1,87 @@
 meaning that it's based on untested ideas or techniques and not yet established
 or finalized or involves a radically new and innovative style! This means that
 support is best effort (at best!) and we strongly encourage you to NOT use this
 in production.
 
+Kubernetes-native declarative infrastructure for Kubernetes clusters on Tinkerbell.
+
 ---
 
-Cluster API Provider Tinkerbell (CAPT) is the implementation of Cluster API
-Provider for Tinkerbell.
+## What is the Cluster API Provider Tinkerbell
 
-## Goal
+The [Cluster API][cluster_api] brings declarative, Kubernetes-style APIs to Kubernetes
+cluster creation, configuration and management.
 
-* It acts as a bridge between Cluster API (a Kubernetes sig-lifecycle project)
-  and Tinkerbell
-* It simplifies Kubernetes cluster management using Tinkerbell as underline
-  infrastructure provider
-* Create, update, delete Kubernetes clusters in a declarative fashion.
+The API itself is shared across multiple cloud providers allowing for true hybrid
+deployments of Kubernetes, both on-premises and off.
 
-## Current state
+---
+
+## Quick Start
+
+See the [Quick Start](docs/QUICK-START.md)
+
+---
+
+## Compatibility with Cluster API and Kubernetes Versions
+
+This provider's versions are compatible with the following versions of Cluster API:
+
+
+|                                    | v1beta1 (v1.0) |
+| ---------------------------------- | -------------- |
+| Tinkerbell Provider v1beta1 (v0.1) | ✓              |
 
-7th December 2020 marks the first commit for this project, it starts as a
-porting from CAPP (cluster api provider packet).
 
-Currently, it is possible to bootstrap both single instance and HA Control Plane workload 
-clusters using hardware managed by Tinkerbell.
+This provider's versions are able to install and manage the following versions of Kubernetes:
+
+|                              | v1.19 | v1.20 | v1.21 | v1.22 |
+| ---------------------------- | ----- | ----- | ----- | ----- |
+| AWS Provider v1beta1 (v0.1)  | ✓     | ✓     | ✓     | ✓     |
+
+\* Not management clusters
+
+Each version of Cluster API for Tinkerbell will attempt to support all community supported Kubernetes versions during it's maintenance cycle; e.g., Cluster API for Tinkerbell `v0.1` supports Kubernetes 1.19, 1.20, 1.21, 1.22 etc.
+
+**NOTE:** As the versioning for this project is tied to the versioning of Cluster API, future modifications to this
+policy may be made to more closely align with other providers in the Cluster API ecosystem.
+
+---
+
+## Kubernetes versions with published Images
+
+pre-built images are pushed to the github container registry. We currently publish images for Ubuntu 18.04 and Ubuntu 20.04.
+
+- [Ubuntu 18.04](https://github.com/tinkerbell/cluster-api-provider-tinkerbell/pkgs/container/cluster-api-provider-tinkerbell%2Fubuntu-1804)
+- [Ubuntu 20.04](https://github.com/tinkerbell/cluster-api-provider-tinkerbell/pkgs/container/cluster-api-provider-tinkerbell%2Fubuntu-2004)
+
+---
```

Comment:
A markdown linter would add a newline after this horizontal-rule, same on line 84.  (I'm not sure if it is to add separation around the horizontal rule, or the header).